parley/src/tests/test_basic.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 the Parley Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use crate::{testenv, Alignment, InlineBox};
+
+#[test]
+fn test_plain_multiline_text() {
```

Comment:
The `test_` prefix is not idiomatic in Rust - see https://github.com/rust-lang/rust-clippy/issues/8931; the same probably also applies to the module name, since it's in the `tests` folder.
```suggestion
fn plain_multiline_text() {
```

Reply:
Fixed